#
# Add yet another possible source for video
# "streamer" can capture from a video4linux device
#
Index: xscreensaver/driver/xscreensaver-getimage-video
===================================================================
--- xscreensaver.orig/driver/xscreensaver-getimage-video	2013-11-23 16:16:12.000000000 +0100
+++ xscreensaver/driver/xscreensaver-getimage-video	2013-11-23 16:16:12.000000000 +0100
@@ -34,6 +34,8 @@
 
 my $tmpdir   = $ENV{TMPDIR} || "/tmp";
 my $tmpfile  = sprintf("%s/xssv.%08x.ppm", $tmpdir, rand(0xFFFFFFFF));
+my $tmpfile1 = sprintf ("%s/xssgv01.ppm", $tmpdir, rand(0xFFFFFFFF));
+my $tmpfile3 = sprintf ("%s/xssgv03.ppm", $tmpdir, rand(0xFFFFFFFF));
 
 my $verbose = 0;
 
@@ -46,7 +48,7 @@
 # If you add other programs to this list, please let me know!
 #
 my @programs = (
-
+  "streamer -a -t3 -r1 -o $tmpfile1; mv $tmpfile3 $tmpfile",		# XawTV after 2 sec
   "bttvgrab -d q -Q -l 1 -o ppm -f $tmpfile",	# BTTV
   "qcam > $tmpfile",				# Connectix Qcam
   "gqcam -t PPM -d $tmpfile",			# GTK+ Qcam clone
